handbook/direction/code-graph/index.md

```diff
@@ -0,0 +1,41 @@
+# Code Graph
+
+The Code Graph organization comprises teams working on search, insights, code intelligence, and batch changes, and this page describes the overarching mission and strategy that ties them together. Each one also has an individual direction page, indexed from our [direction home page](../index.md#code-graph), that describes what they are working on and why in more detail.
+
+## Mission
+
+We build the Code Graph to make working with code easier, regardless of how much you have, how complex it is, where you store it, or even how technical you are.
+
+## Vision
+
+### One year vision
+
+> Within the next year, we want to ensure that we are solving the most critical problems for our enterprise users, focusing on retaining existing and acquiring new enterprise customers. By providing as much value as possible to them, we can grow the business while solving the hardest scaling and complexity problems for everyone.
```

Comment:
```suggestion
> We want to ensure that we are solving the most critical problems for our enterprise users, focusing on retaining existing and acquiring new enterprise customers. By providing as much value as possible to them, we can grow the business while solving the hardest scaling and complexity problems for everyone.
```

Minor – these time bound phrases at the start of the vision sections (three total) are already covered by the subject headings, and IMO it's almost more "shazam!" if we just state it directly without the qualifying phrases, like the toplevel [company vision section does](https://about.sourcegraph.com/handbook/company/strategy#10-year-vision).